docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
You can generate a guid [here](http://guidgenerator.com)

How it is used is described [here](https://github.com/dotnet/docs/blob/d97a1501ad25b683cbb5d7fbd8bd1b137f7f4046/styleguide/template.md#metadata)